src/game/default/g_main.c

```diff
@@ -182,9 +183,16 @@ void G_ResetItems(void) {
 			continue;
 		}
 
+		if (ent->locals.item->type == ITEM_TECH) { // free techs, we'll spawn them again
+			G_FreeEntity(ent);
+			continue;
```

Comment:
Maybe, seems like a very specific feature but I can see it being useful (fight for the center techs).